packages/rolldown/src/utils/compose-js-plugins.ts

```diff
@@ -0,0 +1,276 @@
+import { Plugin, RolldownPlugin } from '../plugin'
+import _ from 'lodash-es'
```

Comment:
This doesn't seem to be used at all?
Also, even if lodash is needed, we should use named import to avoid bundling all its methods.

Reply:
Yes. It's not used🤡. Rolldown supports treeshaking on `_.x`. I guess it would be fine.

Reply:
Yeah, rolldown would delete the whole module if the import is not used at all since `lodash-es` is side effects free.

Reply:
> Yes. It's not used🤡. Rolldown supports treeshaking on `_.x`. I guess it would be fine.

But for your case, it will keep the whole module, since you use default import rather than `import * as _ from 'lodash-es'`, so I advice removing the statement or using `import {add } from 'lodash-es'` or `import * as _ from 'lodash-es'`

Reply:
I tested with `webpack` and `rolldown`